tools/rollup-es3/tsconfig.json

```diff
@@ -7,8 +7,8 @@
     "moduleResolution": "node",
     "target": "es3",
     "forceConsistentCasingInFileNames": true,
-    "importHelpers": true,
-    "noEmitHelpers": true,
+    "importHelpers": false,
+    "noEmitHelpers": false,
```

Comment:
This is the normal approach for breaking the tslib dependency, using this here as we don't have multiple __extend usages, so it's not as much of an issue for this development only tool